engine/src/flutter/fml/message_loop_task_queues.cc

```diff
@@ -17,6 +17,7 @@
 namespace fml {
 
 const size_t TaskQueueId::kUnmerged = ULONG_MAX;
+const size_t TaskQueueId::kInvalid = ULONG_MAX - 1;
```

Comment:
What do you think of calling this something like `kNone` and renaming `TaskQueueId::is_valid` to `TaskQueueId::exists`?

I worry we might be tempted to reuse the "invalid" state for more than "This task runner does not have its own task queue". I don't feel strongly about this, feel free to skip